syntax="proto3";

package IBUS;

message FaultDetectInfo{
  uint32 sub_system = 1;       //故障来源
  uint32 fault_level = 2;      //故障等级
  uint32 dtc = 3;              //故障码
  uint32 fault_id = 4;         //故障索引
}

message FaultDetectMsg{
  uint32 moudle_id = 1;        //模块ID
  repeated FaultDetectInfo detect_info = 2;  //故障信息列表,列表为空表示没有故障
}